feature/request-money/src/main/kotlin/org/mifospay/feature/request/money/ShowQrContent.kt

```diff
@@ -76,7 +74,7 @@ internal fun ShowQrContent(
             ) {
                 Text(
                     text = stringResource(id = R.string.feature_request_money_title),
-                    color = getColor(),
+                    color = MifosBlue,
```

Comment:
Why are we still using MifosBlue? Didn’t we already change the theme?

Reply:
dark theme for this is not provided for applying same figma theme for both dark mode and light mode i have used this

Reply:
I think since we've updated the theme for other screens, we should apply it here as well. Otherwise, it might appear inconsistent.
@niyajali what are your thoughts on this?